fmpz_poly/mullow_SS.c

```diff
@@ -37,8 +37,15 @@ void _fmpz_poly_mullow_SS(fmpz * output, const fmpz * input1, slong len1,
     loglen2 = FLINT_CLOG2(len2);
     n = (WORD(1) << (loglen - 2));
 
-    size1 = _fmpz_vec_max_limbs(input1, len1); 
-    size2 = _fmpz_vec_max_limbs(input2, len2);
+    bits1 = _fmpz_vec_max_bits(input1, len1);
```

Comment:
I don't understand why it is better to use the presumably more expensive vec_max_bits function given that we convert it to a number of limbs below. The actual bits are only used to determine if ant coefficients were signed right? But we get that info from get_fft below, so where is the saving?

Reply:
Nevermind, I see that the get_fft function now no longer needs to return the sign. Yeah this is fine.